gender_analysis/common.py

```diff
@@ -163,8 +162,8 @@ def get_text_file_encoding(filepath):
     >>> from gender_analysis import common
     >>> text = 'here is an ascii text'
     >>> file_path = Path(common.BASE_PATH, 'example_file.txt')
-    >>> with codecs.open(file_path, 'w', 'utf-8') as source:
-    ...     source.write(text)
+    >>> with open(file_path, 'w') as source:
```

Comment:
Unless specified, `open()` uses the platform's default encoding, which on Windows is, alas, not utf-8. If you could add `encoding='utf-8'`here, this looks good to go!

(Here it actually doesn't matter since these are all ASCII characters, but just for the sake of consistency through the project...)

Reply:
Thanks for pointing that out - I've specified `utf-8` now. (About time Windows catches up!)